data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -96,8 +96,10 @@ podman run -d --net host --privileged --name httpd \
 
 {{ if .PlatformData.BareMetal.ProvisioningIPv6 }}
 IPTABLES=ip6tables
+IP_OPTIONS="ip=dhcp6"
```

Comment:
We need to set this if the API VIP (where the MCS download happens) is v6, not if the provisioning network is v6 - there's a template data field we can probably use:

https://github.com/openshift/installer/blob/master/pkg/asset/ignition/bootstrap/common.go#L297

https://github.com/openshift/installer/blob/master/pkg/asset/ignition/machine/node.go#L36